additional_content.md

```diff
@@ -160,16 +160,17 @@ The samples SHOULD be written to the Dataset file without separation, and the
 Dataset file MUST NOT contain any other characters (e.g., delimiters,
 whitespace, line-endings, EOF characters).
 
-Complex samples MUST be interleaved, with the in-phase component first (i.e.,
-`I[0]` `Q[0]` `I[1]` `Q[1]` ... `I[n]` `Q[n]`). When `core:num_channels` in the
-Global Object (described below) indicates that the Recording contains more than one channel,
+Complex samples MUST be interleaved, with the in-phase component first (i.e., `I[0]` `Q[0]` `I[1]` `Q[1]` ... `I[n]` `Q[n]`). 
+
+When `core:num_channels` in the Global Object (described below) indicates that the Recording contains more than one channel,
+which is intended for use in non-RF application such as audio or oscilloscope channels, 
 samples from those channels MUST be interleaved in the same manner, with
 the same index from each channel's sample serially in the Recording. For
 example, a Recording with two channels of `ri16_le` representing real-valued
 audio data from a stereo Recording and here labeled `L` for left and `R` for
 right, the data MUST appear as `L[0]` `R[0]` `L[1]` `R[1]` ... `L[n]` `R[n]`.
-The data type specified by `core:data_type` applies to all channels of data
-both real and imaginary parts.
+The data type specified by `core:data_type` applies to all channels of data both real and imaginary parts. 
+For multiple channels of IQ data, it is RECOMMENDED to use SigMF Collections instead of num_channels for widest application support.
```

Comment:
Recommendations:

> The data type specified by `core:data_type` applies to all channels of data. 

the whole point of this change is to suggest AGAINST using this for real/imag data so lets remove this awkward language. Also it makes no sense for recommended (non-I/Q) usage.

> When recording multiple channels of complex data, SigMF Collections are STRONGLY RECOMMENDED over multiple channels of data in a single data file. This will provide the widest application support, and allows for separate captures and annotation scope metadata.